src/site/content/en/blog/media-session/index.md

```diff
@@ -0,0 +1,405 @@
+
+navigator.mediaSession.setActionHandler("seekbackward", (details) => {
+  const skipTime = details.seekOffset || defaultSkipTime;
+  video.currentTime = Math.max(video.currentTime - skipTime, 0);
+  // TODO: Update playback state.
+});
+
+navigator.mediaSession.setActionHandler("seekforward", (details) => {
+  const skipTime = details.seekOffset || defaultSkipTime;
+  video.currentTime = Math.min(video.currentTime + skipTime, video.duration);
+  // TODO: Update playback state.
+});
+```
+
+### Seek to a specific time
+
+The "seekto" action indicates that the user wants to move the media playback
+time to a specific time.
+
+The `seekTime` value provided in the action handler is the time in seconds to
+move the media playback time to.
+
+The `fastSeek` boolean provided in the action handler is true if the action is
+being called multiple times as part of a sequence and this is not the last call
+in that sequence.
+
+```js
+const video = document.querySelector('video');
+
+navigator.mediaSession.setActionHandler("seekto",(details) => {
+  if (details.fastSeek && "fastSeek" in video) {
+    // Only use fast seek if supported.
+    video.fastSeek(details.seekTime);
+    return;
+  }
+  video.currentTime = details.seekTime;
+  // TODO: Update playback state.
+});
+```
+
+## Set playback position
+
+Accurately displaying the media playback position in a notification is as simple
+as setting the position state at an appropriate time as shown below. The
+position state is a combination of the media playback rate, duration, and
+current time.
+
+{% Aside 'gotchas' %}
+The position state is supported on Android as of Chrome 81 and later.
+{% endAside %}
+
+```js
+const video = document.querySelector("video");
+
+function updatePositionState() {
+  if ("setPositionState" in navigator.mediaSession) {
+    navigator.mediaSession.setPositionState({
+      duration: video.duration,
+      playbackRate: video.playbackRate,
+      position: video.currentTime
+    });
+  }
+}
+
+// When video starts playing, update duration.
+await video.play();
+updatePositionState();
+
+// When user wants to seek backward, update position.
+navigator.mediaSession.setActionHandler("seekbackward", details => {
+  /* ... */
+  updatePositionState();
+});
+
+// When user wants to seek forward, update position.
+navigator.mediaSession.setActionHandler("seekforward", details => {
+  /* ... */
+  updatePositionState();
+});
+
+// When user wants to seek to a specific time, update position.
+navigator.mediaSession.setActionHandler("seekto", details => {
+  /* ... */
+  updatePositionState();
+});
+
+// When video playback rate changes, update position state.
+video.addEventListener("ratechange", event => {
+  updatePositionState();
+});
+```
+
+Resetting the position state is as easy as setting it to `null`.
+
+```js
+// Reset position state when media is reset.
+navigator.mediaSession.setPositionState(null);
+```
+
+## Support
```

Comment:
Please rename this to `Cross-browser support` and make it the first section of the article.

I updated the checklist to include this guidance: https://github.com/GoogleChrome/web.dev/commit/7d765fa5e7b9758033890b5553482062e534e823#diff-42b91fa51d3330a54bb6e369d4e32982

Reply:
Done